db/migrate/20180524190818_destroy_non_existing_git_hub_users.rb

```diff
@@ -1,6 +1,14 @@
 class DestroyNonExistingGitHubUsers < ActiveRecord::Migration[5.1]
+
   def up
-    unless ENV['RAILS_ENV'] == 'development'
+    return if ENV['RAILS_ENV'] == 'test'
+    if ENV['RAILS_ENV'] == 'development'
+      puts "Updating users with github_id ...."
+      users = User.all.select { |user| user.github_id.nil? }
+      users.each_with_index do |user, index|
+        user.update!(github_id: index + 10)
+      end
```

Comment:
@F3PiX `select` is doing this in ruby which is slow 😟 `users = User.where(github_id: nil)` does the same thing but in the database, which is fast!

`users.each_with_index` is not great for memory because it loads every record without a github id at the same time into memory.

Here's an alternative:

```ruby
User.where(github_id: nil).find_each(batch_size: 200).with_index do |user, index|
  user.update!(github_id: index + 10)
end
```

You can find a similar example in the docs here: http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each